package.json

```diff
@@ -0,0 +1,111 @@
+{
+  "name": "react-targem",
+  "version": "0.0.0",
+  "description": "A React library for efficiently implementing Gettext localization",
+  "main": "./dist/bundles/index.umd.js",
+  "module": "./dist/esm5/index.js",
+  "es2015": "./dist/esm2015/index.js",
+  "typings": "./dist/types/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/trucknet-io/react-targem.git"
+  },
+  "keywords": [
+    "react",
+    "targem",
+    "i18n",
+    "typescript",
+    "localization"
+  ],
+  "author": "goooseman, alexeychikk",
+  "license": "MIT",
+  "scripts": {
```

Comment:
need to write `prepublishOnly` script, which will build the app

Reply:
@goooseman I use `prerelease` so that `bundle-sizes.lock` file is commited.